app/screens/home-screen/home-screen.tsx

```diff
@@ -145,8 +150,21 @@ export const HomeScreen: React.FC = () => {
     }
   }, [isAuthed, refetchAuthed, refetchRealtimePrice, refetchUnauthed])
 
-  const transactionsEdges =
-    dataAuthed?.me?.defaultAccount?.transactions?.edges ?? undefined
+  const pendingTransactions = dataAuthed?.me?.defaultAccount?.pendingTransactions
+  const transactionsEdges = dataAuthed?.me?.defaultAccount?.transactions?.edges
+
+  const transactions = useMemo(() => {
+    const transactions: TransactionFragment[] = []
+    if (pendingTransactions) {
+      transactions.push(...pendingTransactions)
+    }
+    const settledTransactions =
+      transactionsEdges
+        ?.map((edge) => edge.node)
+        .filter((tx) => tx.status !== TxStatus.Pending) ?? []
```

Comment:
are we going to remove this filter after the use have updated to the new app?
because we can then remove the pending tx from the "main" tx query?

Reply:
We need this deployed and then we can fix the backend to remove pending transactions from the transactions query and then we can remove this filter.